docker/docker-compose.yml

```diff
@@ -28,7 +25,7 @@ x-php: &php
     PAGER: more
     HM_ENV_ARCHITECTURE: 'local-server'
     HM_DEPLOYMENT_REVISION: 'dev'
-    ELASTICSEARCH_HOST: elasticsearch
+    ELASTICSEARCH_HOST: elasticsearch.${COMPOSE_PROJECT_NAME:-default}.altis.dev
     ELASTICSEARCH_PORT: 9200
     AWS_XRAY_DAEMON_HOST: xray
     S3_UPLOADS_ENDPOINT: http://s3.localhost:8000/
```

Comment:
@joehoyle 	I can't find any reference to `S3_UPLOADS_ENDPOINT` in S3 Uploads. How/where is this used? I'm working on updating references to services in a way that won't cause issues like #8 again.

Reply:
It's not automatically used, see https://github.com/humanmade/local-server/blob/master/load.php#L26 for an example of it being used though

Reply:
Got it, thanks for that!

Reply:
That constant has been requested a fair bit on S3 Uploads, we may as well support it natively at this point.